oidc-signature-extension.md

```diff
@@ -262,200 +208,224 @@ The following is a non-normative example of the claims in a Request Object befor
+**Note:** The `https://id.oidc.se/scope/sign` alone does not say anything about the identity of the signing end-user.
+A Relying Party wishing to get this information, which it most likely does, should include additional scopes in the 
+request that declares which identity claims that are wanted. 
 
-| Claim | Description/comment | Requirement |
-| :--- | :--- | :--- |
-| `https://claims.oidc.se/`<br />`1.0/userSignature` | The user signature. For federated signing a user signature MAY be created if the identity provider supports signing. | Optional |
-| `https://claims.oidc.se/`<br />`1.0/signMessageDigest` | A digest of the sign message that was sent by the client in the request. This claim MUST be included iff the client included a sign message in the request and this message was display for, and approved by, the user.  | Optional |
-| `https://claims.oidc.se/1.0/sad` | Signature activation data.<br />*TODO: More will be written about the conditions of including the sad claim.* | Optional |
 
 <a name="relying-party-requirements"></a>
 ## 4. Relying Party Requirements
 
 <a name="requests"></a>
 ### 4.1. Requests
 
-TODO: Refer to iGov, or our own main profile regarding requirements for request parameters.
+A Relying Party wishing to issue a request for signature according to the specification MUST include 
+`https://id.oidc.se/scope/sign` along with the mandatory `openid` as values to the `scope` request parameter. 
 
-A Relying Party wishing to issue a request for signature MUST include either the `https://scopes.oidc.se/1.0/delegatedSign` scope or the `https://scopes.oidc.se/1.0/federatedSign` scope along with the mandatory `openid` scope as values to the `scope` request parameter. It MUST NOT include both signing scopes.
+A request for signature MUST contain the [Signature Request Parameter](#the-signature-request-parameter) and its
+inclusion in the request MUST follow the requirements stated in sections [3.1.1](#placement-of-the-parameter-in-an-authentication-request), [Placement of the Parameter in an Authentication Request](#placement-of-the-parameter-in-an-authentication-request) and [3.1.2](#security-requirements), [Security Requirements](#security-requirements).
 
-A request for signature MUST contain the [Signature Request Parameter Claim](#the-signature-request parameter-claim) and its inclusion in the request MUST follow the requirements stated in sections [3.2.1](#placement-of-the-parameter-in-an-authentication-request), "[Placement of the Parameter in an Authentication Request](#placement-of-the-parameter-in-an-authentication-request)" and [3.2.2](#security-requirements), "[Security Requirements](#security-requirements)".
+The authentication request MUST contain the `prompt` parameter<sup>1</sup> and its value MUST be include both the 
```

Comment:
Nitpick: "its value MUST be include both the ... " should be "its value MUST include both the ... "